docs/sources/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ The CHANGELOG for the current development version is available via
 ##### New Features
 
 - New `ExhaustiveFeatureSelector` estimator in `mlxtend.feature_selection` for evaluating all feature combinations in a specified range
+- New 'StackingCVClassifier' estimator in 'mlxtend.classifier' for implementing a stacking ensemble that uses cross-validation techniques for training the meta-estimator to avoid overfitting.
```

Comment:
I'd suggest you add your name or GitHub handle to this line. E.g., [Reiichiro Nakano](https://github.com/reiinakano)